efficientdet/keras/postprocess.py

```diff
@@ -24,13 +24,19 @@
 from keras import anchors
 T = tf.Tensor  # a shortcut for typing check.
 CLASS_OFFSET = 1
+# TFLite-specific constants.
+TFLITE_MAX_CLASSES_PER_DETECTION = 1
+TFLITE_DETECTION_POSTPROCESS_FUNC = 'TFLite_Detection_PostProcess'
+# TFLite fast NMS == postprocess_global (less accurate)
+# TFLite regular NMS == postprocess_per_class
+TFLITE_USE_REGULAR_NMS = False
 
 
 def to_list(inputs):
   if isinstance(inputs, dict):
-    return [tf.cast(inputs[k], tf.float32) for k in sorted(inputs.keys())]
+    return [inputs[k] for k in sorted(inputs.keys())]
   if isinstance(inputs, list):
-    return [tf.cast(i, tf.float32) for i in inputs]
```

Comment:
I remember it fixs mixed precision eval .

Reply:
Done.

Reply:
```# TODO(tanmingxing): remove this cast once FP16 works postprocessing.```
The comment should be added here?

Reply:
I think if we do the conversion right after the network, then we don't need these casts any more.

Reply:
https://github.com/google/automl/blob/539ab6554c4b47e319a43da29125714bb4adfe35/efficientdet/keras/train_lib.py#L339
If so, please add cast here too.

Reply:
Done.